subscription/index.md

```diff
@@ -72,9 +72,9 @@ Please read the [Blog](https://www.docker.com/blog/updating-product-subscription
 
 > **Note**
 >
-> The licensing and distribution terms for Docker and Moby **open-source** projects, such as Docker Engine, aren't changing.
+> The licensing and distribution terms for Docker and Moby open-source projects, such as Docker Engine, aren't changing.
 
 Docker Desktop is built using open-source software. For information about the licensing of open-source components in Docker Desktop, Select ![whale menu](../desktop/mac/images/whale-x.svg){: .inline} > **About Docker Desktop** > **Acknowledgements**.
 
-Docker Desktop distributes some components that are licensed under the
+Docker Desktop distributes some components that's licensed under the
```

Comment:
I think this should be `are` because the object, `components`, is plural.
```suggestion
Docker Desktop distributes some components that are licensed under the
```

Reply:
Thanks Craig! I changed it to "are", but the Vale rule suggested "that's" to keep it active voice

Reply:
updated